woocommerce/woocommerce-android
# [Customer Search] Change empty search screen

Closes: #9491


### Description

* Initial empty state of the screen on the old WC version is changed. Now there is a button that leads to the customer details
* Search filed hints for old and new WC versions are different
* Search field is getting focused right away on the old WC version

### Testing instructions

Run customer search on both 8- and 8+ WC version and notice the changes listened in the description

### Images/gif


https://github.com/woocommerce/woocommerce-android/assets/4923871/41baa8b8-0c6e-4638-bc47-64915102af04
